andela/nazgul-rc
# Feature/#157188286/ratings and reviews

#### What does this PR do?
Integrates product ratings and reviews into app

#### Description of Task to be completed?

- Allow a user to rate a product
- Allow a user to edit his/her rating for a product
- Allow a user to review a product
- Allow a user to submit review for a product

#### How should this be manually tested?

- npm run git clone `https://github.com/andela/nazgul-rc.git` to clone the repository
- navigate to the nazgul-rc folder
- run `curl https://install.meteor.com/ | sh`
- run `npm install -g reaction-cli`
- install dependencies with`meteor npm install`
- run `reaction` to start up server
- once app is running, navigate to the app using localhost:3000
- navigate to shop page
- click on a product
- see average rating for that product
- see reviews for that product
- rate and review the product

#### What are the relevant pivotal tracker stories?
#157188286
